test/java/cucumbertests/IceFieldTests.java

```diff
@@ -0,0 +1,85 @@
+package cucumbertests;
+
+import Field.IceBlock;
+import Player.Eskimo;
+import Player.Researcher;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class IceFieldTests {
+    IceBlock field = new IceBlock();
+    List<Eskimo> eskimos = new ArrayList<>();
+    List<Researcher> researchers = new ArrayList<>();
+    
+    @io.cucumber.java.en.Given("^a stable ice field$")
+    public void aStableIceField() {
+        field.setCapacity(-1);
+    }
+
+    @io.cucumber.java.en.Given("^(\\d+) eskimos$")
+    public void eskimos(int numOfEskimos) {
+        for (int i = 0; i < numOfEskimos; i++) {
+            eskimos.add(new Eskimo());
+        }
+    }
+
+    @io.cucumber.java.en.Given("^(\\d+) researchers$")
+    public void researchers(int numOfResearchers) {
+        for (int i = 0; i < numOfResearchers; i++) {
+            researchers.add(new Researcher());
+        }
+    }
+
+    @io.cucumber.java.en.When("^the eskimos move to the field$")
+    public void theEskimosMoveToTheField() {
+        for (var eskimo: eskimos) {
+            field.Accept(eskimo);
+        }
+    }
+
+    @io.cucumber.java.en.And("^the researchers move to the field$")
+    public void theResearchersMoveToTheField() {
+        for (var researcher: researchers) {
+            field.Accept(researcher);
+        }
+    }
+
+    @Then("none of the eskimos should be in water")
+    public void noneOfTheEskimosShouldBeInWater() {
+        for (var e : eskimos) {
+            if (e.IsInWater()) fail();
```

Comment:
A javítások eredménye képpen azt hiszem az IsInWater() függvény át lett nevezve isInWater()-re, ezt javítsd légyszi. (Ha vissza mergeled a main branchet ebbe, akkor úgy is hibát fog dobni az IDE)